### DISCUSSION TITLE
Almost had a heart attack, no modeline ??? order restored with doom-modeline

### DISCUSSION THREAD

**Discussion by ghost** [Lambda-Emacs/lambda-emacs#19] [General]
how awesome is this
![image](https://user-images.githubusercontent.com/89839215/182498075-688b65fa-6c10-4767-bb04-7036da2f4235.png)

**Comment by ghost** [comment]
the dashboard config is different, the original dashboard clicking on files did not work and had two times the recent files showing.

Thank you for your time, forgive my foolishness I was so impatient but now I have settled for good.

my dashboard is this 
https://github.com/wereket/lambda-emacs/blob/main/.local/lambda-library/lambda-setup/lem-setup-dashboard.el

Thank you for your time and awesome config.

**Comment by mclearc** [comment]
Glad it worked! And please feel free to join the discord (I started one today — you can find the link from the readme)

--
Colin McLear
Associate Professor
Department of Philosophy 
University of Nebraska–Lincoln
colinmclear.net
